examples/tcp_server.rs

```diff
@@ -1,19 +1,17 @@
 // You can run this example from the root of the mio repo:
 // cargo run --example tcp_server --features="os-poll net"
-use mio::event::Event;
```

Comment:
Can you revert all changes here, the changes make the example much harder to read. Please use the same structure as for the UDP example, with the extra `main`. I know it will generate a number of warnings, but that's OK.

Reply:
yeah, will do